### PR TITLE
EMP-198: Adding S3 bubkcet permissions to list buckert and objects

### PR DESCRIPTION
**Pull request by tanovellino**

### CODE REVIEW

File: namespaces/live.cloud-platform.service.justice.gov.uk/laa-crime-equinity-historical-data-prod/resources/s3.tf [hunk 1]
```diff
@@ -109,44 +109,47 @@ module "s3_bucket" {
...
-        "AWS": "arn:aws:iam::012345678901:user/foobar"
+  bucket_policy = <<EOF
+  {
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Sid": "AllowBucketAccess",
+        "Effect": "Allow",
+        "Principal": {
+          "AWS": "${module.s3_bucket.bucket_arn}"
```
Comment (jaskaransarkaria) [inline namespaces/live.cloud-platform.service.justice.gov.uk/laa-crime-equinity-historical-data-prod/resources/s3.tf:122]: this needs to be your user arn

File: namespaces/live.cloud-platform.service.justice.gov.uk/laa-crime-equinity-historical-data-prod/resources/s3.tf [hunk 2]
```diff
@@ -109,44 +109,47 @@ module "s3_bucket" {
...
+        },
+        "Action": [
+          "s3:ListBucket",
+          "s3:GetBucketLocation",
+        ],
+        "Resource": [
+          "${module.s3_bucket.bucket_arn}",
```
Comment (jaskaransarkaria) [inline namespaces/live.cloud-platform.service.justice.gov.uk/laa-crime-equinity-historical-data-prod/resources/s3.tf:129]: add another resource like this `"${module.s3_bucket.bucket_arn}/*"`

**Review by jaskaransarkaria — APPROVED** [review]

File: namespaces/live.cloud-platform.service.justice.gov.uk/laa-crime-equinity-historical-data-prod/resources/s3.tf (hunk 2 above)
Comment (tanovellino) [reply namespaces/live.cloud-platform.service.justice.gov.uk/laa-crime-equinity-historical-data-prod/resources/s3.tf]: addressed on refactored version

File: namespaces/live.cloud-platform.service.justice.gov.uk/laa-crime-equinity-historical-data-prod/resources/s3.tf (hunk 1 above)
Comment (tanovellino) [reply namespaces/live.cloud-platform.service.justice.gov.uk/laa-crime-equinity-historical-data-prod/resources/s3.tf]: fixed

**Review by jaskaransarkaria — APPROVED** [review]

**Pull request merged**